resources/data/default_files.json

```diff
@@ -0,0 +1,105 @@
+{
+	"addons":{
+		"root":"special://home/addons",
+		"dirs":[
+			{
+				"type":"include",
+				"path":"special://home/addons",
+				"recurse":true
+			},
+			{
+				"type":"exclude",
+				"path":"special://home/addons/packages"
+			},
+			{
+				"type":"exclude",
+				"path":"special://home/addons/temp"
+			}
+		]
+	},
+	"addon_data":{
+			"root":"special://home/userdata/addon_data",
+			"dirs":[
+			{
+				"type":"include",
+				"path":"special://home/userdata/addon_data",
+				"recurse":true
+			}
+		]
+	},
+	"database":{
+			"root":"special://home/userdata/Database",
+			"dirs":[
+			{
+				"type":"include",
+				"path":"special://home/userdata/Database",
+				"recurse":true
+			}
+		]
+	},
+	"game_saves":{
```

Comment:
worth pointing out for documentation purposes that this is specific to Leia+ versions of Kodi where the Retroplayer branch has been merged in.